druid/src/event.rs

```diff
@@ -265,6 +265,13 @@ pub enum LifeCycle {
     /// [`Size`]: struct.Size.html
     /// [`Widget::layout`]: trait.Widget.html#tymethod.layout
     Size(Size),
+    /// Called when the Disabled state of the widgets is changed.
+    ///
+    /// [`is_disabled`](struct.EventCtx.html#method.is_disabled) returns if the widget is disabled.
+    ///
+    /// [`set_disabled`](struct.EventCtx.html#method.set_disabled) to change the widgets disabled
+    /// state.
```

Comment:
Project style is to avoid inline links, it makes it harder to read the docs when you're in the actual code.
```suggestion
    /// To check if a widget is disabled, see [`is_disabled`].
    ///
    /// To change a widget's disabled state, see [`set_disabled`].
    ///
    /// [`is_disabled`]: crate::EventCtx::is_disabled
    /// [`set_disabled`]: crate::EventCtx::set_disabled
```